src/utils/find.ts

```diff
@@ -0,0 +1,13 @@
+import { Account, EnvRegExp } from '../project';
+
+export function findAccountTypeByEnvironment(obj: Record<string, Account>, predicate: string): string | undefined {
+  for (const key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      const list = obj[key].environments?.filter((environment) => new EnvRegExp(environment).test(predicate));
```

Comment:
Could this be written using `some` instead of `filter`?

Reply:
Actually it can ... thank you to pointed that out.